packages/aws-cdk-lib/aws-rds/lib/private/util.ts

```diff
@@ -169,3 +172,31 @@ export function applyDefaultRotationOptions(options: CommonRotationUserOptions,
     ...options,
   };
 }
+
+/**
+ * Determines the value of the `storageEncrypted` property for a cluster or instance.
+ */
+export function getStorageEncryptedProperty(scope: IConstruct, props: DatabaseInstanceProps | DatabaseClusterProps): boolean | undefined {
```

Comment:
```suggestion
export function getStorageEncryptedProperty(
  scope: IConstruct, 
  storageEncrypted?: boolean, 
  storageEncryptedLegacyDefaultValue?: boolean, 
  storageEncryptionKey?: kms.IKey
): boolean | undefined {
```
Parameters should be specified explicitly as using union types is against [design guidelines](https://github.com/aws/aws-cdk/blob/main/docs/DESIGN_GUIDELINES.md#unions).